tests/test_model.py

```diff
@@ -1334,26 +1369,28 @@ def test_bad_explicit_model_implicit_data(train_val_implicit_sample_data):
 def test_get_item_preds_err(implicit_model,
                             train_val_implicit_data):
     model = implicit_model
-    train, val = train_val_implicit_data
+    train, _ = train_val_implicit_data
 
     with pytest.raises(
         ValueError,
-        match=f'``user_id`` 11111 is not in the model. Expected id between 0 and {train.num_users}'
+        match=f'``user_id`` {train.num_users + 1} is not in the model. '
```

Comment:
[Optional] It might be nice to make this a variable that you use twice in this test, something like: 

```python
user_id_who_does_not_exist = train.num_users + 1
```

Reply:
`nonexistent_user_id`?

Reply:
Yup, much better hahaha.